pipelines/README.md

```diff
@@ -61,133 +61,159 @@ lightgbm-mushrooms/
 - A clone of this repository
 
-## Deploy build pipeline for AI runtime container images
+## AI Edge End to End Pipline
 
-### Setup S3 bucket and credentials
+### Description
+End to end pipeline that supports a workflow to Fetch -> Build -> Test -> Push an immutable model container image to an image registry
+
+* Fetch - Fetch a pre-trained [OpenVino](https://docs.openvino.ai/2023.2/Supported_Model_Formats.html) or [Seldon MLServer](https://mlserver.readthedocs.io/en/latest/runtimes/index.html) compatible model from S3 or a Git repo
+* Build - Package and build the model image and it's dependencies into an immutable container image
+* Test - Run a smoke test against a Pod running the immutable container to verify the exposed endpoint is available for inferencing
+* Push - Push the immutable container image to an image registry supported
+
+### Setup S3 bucket
 
 Create an S3 bucket and upload the directories with the models:
 
 ![S3 models example](../.github/images/S3-models.png)
 
-Fill information about access to your S3 bucket in a copy of [`aws-env.yaml`](tekton/build-container-image-pipeline/aws-env.yaml).
 If you don't have or know your access key, generate one in AWS account's Security credentials > Access keys.
 
-Then store the credentials in an OpenShift secret:
+### Quay Repository and Robot Permissions
+- In your image registry namespace ([Quay](https://quay.io)):
+  - Add a robot account to push images and set write Permissions for the robot account on the repositories. ([Quay](https://access.redhat.com/documentation/en-us/red_hat_quay/3.10/html/use_red_hat_quay/use-quay-manage-repo))
+  - Take note of the username and password of the robot account.
+- Another option is to directly download the Kubernetes Secret of the robot account and store it in a YAML file.
+  - Inspect the file with the pull secret and note the name of the secret, or edit it.
+  - Create the secret and link it to the `pipeline` Service Account that was created by the Red Hat OpenShift Pipelines operator using a Tekton Config. E.g.:
+
+  ```bash
+  oc apply -f <downloaddir>/rhoai-edge-build-secret.yml
+  oc secret link pipeline rhoai-edge-build-pull-secret
+  ```
+
+### Setup credentials
+Create a copy of the file(s) below to include the required credentials for accessing any models and image registries required for the Pipeline to run successfully:
+* S3 Storage - [credentials-s3.secret.yaml.template](tekton/aiedge-e2e/templates/credentials-s3.secret.yaml.template) to include your credentials required to access any model stored in S3 then apply it to the server
+* Image Registry - [credentials-image-registry.secret.yaml.template](tekton/aiedge-e2e/templates/credentials-image-registry.secret.yaml.template) to include the username and password with write access to the image repository.
+  This is needed only in case you noted username and password of the robot account. In case you directly downloaded and applied a Kubernetes Secret, this file can be skipped. 
```

Comment:
Which is the supported way here? `basic-auth` or `dockerconfigjson` secret? Here I mentioned both but the PR check uses the `basic-auth` method. Probably it doesn't matter as both ways are supported by Tekton.

Reply:
I think the default Tekton instructions follow the basic-auth route but `dockerconfigjson` is the easier workflow given that quay.io will provide a [kubernetes secret](https://access.redhat.com/documentation/en-us/red_hat_quay/3.3/html/use_red_hat_quay/use-quay-manage-repo#allow-robot-access-user-repo) with the `.dockerconfigjson` file that is supported by OpenShift Pipelines

Reply:
OK, I will leave the docs mentioning both and in the PR check we will use the `basic-auth`. If that is OK.